onnxruntime/contrib_ops/cuda/activation/activations_impl.cu

```diff
@@ -50,6 +50,17 @@ struct OP_Gelu<half> : public CtxGelu {
   }
 };
 
+template <typename T>
+struct OP_QuickGelu : public CtxQuickGelu {
+  __device__ __inline__ T operator()(const T& a) const {
+    T v = a * static_cast<T>(alpha);
+    T one = static_cast<T>(1.f);
+    T zero = static_cast<T>(0.f);
+    T sigmoid = v >= zero ? one / (one + _Exp(-v)) : one - one / (one + _Exp(v));
```

Comment:
mathematically, both sides of the ternary operator evaluate to the same. What is the reason for the ternary operator here?

Reply:
1/(1+exp(-v)) and 1-1/(1+exp(v)), is it same?

Reply:
yes. mathematically, they are the same. I am not sure if there is another reason for performing the computation this way.

Reply:
Right... They are same. Not sure the reason, just followed what Sigmoid kernel does.